default.nix

```diff
@@ -12,7 +12,7 @@
 , runTests ? false
 # forces naersk (helper tool for building rust projects under nix) to build in a single derivation, instead default way that uses deps and project derivations
 # it is used for building aleph-runtime (we don't want its dependencies to be build separately for a non-WASM architecture)
-, singleStep ? false
+, singleStep ? true
```

Comment:
Two step build (only dependencies -> whole project with prebuilt deps) is currently broken due to some `cargo` resolution magic. This is a workaround.

Reply:
You probably should change the comment as well?

Reply:
Just added a `FIXME` tag for it.

Reply:
That doesn't seem like enough information to understand what this was about even in a month. <_< But since you are essentially the only person who knows what's going on here anyway I guess you can decide, so feel free to resolve if you feel this is sufficient.

Reply:
Added more details. But anyway, running it with this flag set to `true` gives enough info.